lib/b2b/passwords_session.ts

```diff
@@ -23,44 +23,8 @@ export interface B2BPasswordsSessionResetRequest {
   session_token?: string;
   // The JSON Web Token (JWT) for a given Stytch Session.
   session_jwt?: string;
-  /**
-   * Set the session lifetime to be this many minutes from now. This will start a new session if one doesn't
-   * already exist,
-   *   returning both an opaque `session_token` and `session_jwt` for this session. Remember that the
-   * `session_jwt` will have a fixed lifetime of
-   *   five minutes regardless of the underlying session duration, and will need to be refreshed over time.
-   *
-   *   This value must be a minimum of 5 and a maximum of 527040 minutes (366 days).
-   *
-   *   If a `session_token` or `session_jwt` is provided then a successful authentication will continue to
-   * extend the session this many minutes.
-   *
-   *   If the `session_duration_minutes` parameter is not specified, a Stytch session will be created with a
-   * 60 minute duration. If you don't want
-   *   to use the Stytch session product, you can ignore the session fields in the response.
-   */
```

Comment:
Are we ok with getting rid of all this?

Reply:
Hmm, it looks like docs for these fields were never published. I'm not sure how they appeared in here before...